docs/source/using/managing-minishift.adoc

```diff
@@ -104,7 +73,7 @@ places all runtime state into `~/.minishift`. This environment variable is
 currently experimental and semantics might change in future releases.
 
 [[persistent-configuration]]
-=== Persistent configuration
+== Persistent configuration
```

Comment:
persistent options are part of runtime options. the runtime section starts with the four ways to control the runtime behaviour. Then we have three sub headings, one for each of these ways (the 4th is just default value). So I don't understand why this section should move up one level.